tests/framework/installer/install_helper.go

```diff
@@ -341,6 +337,8 @@ func NewK8sRookhelper(clientset *kubernetes.Clientset, t func() *testing.T) *Ins
 	if err != nil {
 		panic("failed to get kubectl client :" + err.Error())
 	}
+	env := objects.E
```

Comment:
why not just use `objects.E` where it's needed instead of declaring the local var `env`?